pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -194,8 +192,6 @@ def advance(self, *args: Any, **kwargs: Any) -> None:
             with self.trainer.profiler.profile("run_training_batch"):
                 batch_output = self.batch_loop.run(batch, batch_idx)
 
-        self.trainer._results.batch_size = batch_size
```

Comment:
reviewer: this is not required since we don't keep track of it now. if it's needed, it will be extracted using the `current_batch` attached to `ResultCollection`

Reply:
It was added by @carmocca as a fix in #9791 (checked git blame) so I am rather confused by your comment 😅

Reply:
it's changed a bit now.. we do keep track of it inside resultCollection. For tbptt I checked and found we don't split it across the batch_size dimension, rather we do it across time dimension